frontend/javascripts/admin/auth/registration_view.js

```diff
@@ -4,15 +4,12 @@ import { Link, useHistory } from "react-router-dom";
 import { Spin, Row, Col, Card } from "antd";
 import messages from "messages";
 import Toast from "libs/toast";
-import { getOrganization, getDefaultOrganization } from "admin/admin_rest_api";
+import { getDefaultOrganization } from "admin/admin_rest_api";
 import features from "features";
-import RegistrationForm from "./registration_form";
+import SpotlightRegistrationForm from "admin/auth/spotlight_registration_form";
+import RegistrationForm from "admin/auth/registration_form";
 
-type Props = {
-  organizationName?: string,
-};
-
-function RegistrationView({ organizationName }: Props) {
+function RegistrationViewNotDemo() {
```

Comment:
I was confused how invite links could still work, although no organization name could be supplied, here, but I found out that for invites the AuthenticationModal is used. Glad you created https://github.com/scalableminds/webknossos/issues/5096 :+1: